configs/helm/README.md

```diff
@@ -0,0 +1,66 @@
+# Telliot
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
+
+A Helm chart for installing Telliot on Kubernetes
+
+## Usage
+
+```bash
+export INSTANCE_NAME=lat
+helm upgrade --install $INSTANCE_NAME . \
+    --namespace tellor --create-namespace 
+```
+
+## Telliot Configuration
+
+Include Telliot configuration files in the files directory of this chart.
+
+These files should be:
+
+- .env
+- config.json
+- configRemote.json
+- configTellorAccess.json
```

Comment:
Telliot needs only a single config file and not 3. It will even run without a config file and will use the defaults.
The config folder contains 3 different config files only as an example on how to run with different settings.

Reply:
Sounds good, I'll adjust the configmap